docs/contributing/performance.md

```diff
@@ -0,0 +1,33 @@
+# Performance Considerations
+
+1. **Profiling with Pyroscope:** Tracee integrates with Pyroscope for continuous profiling. When running Tracee locally for development or testing, use the --pyroscope command-line option. For example:
+
+    ```bash
+    sudo ./dist/tracee --pyroscope
+    ```
+
+    This enables profiling data to be sent to a local Pyroscope server. The Tracee repository includes a convenient way to deploy a performance dashboard for analyzing this data. Run the following for more details:
+
+    ```bash
+    make -f builder/Makefile.performance help
+    ```
+
+2. **Performance Dashboard:** The provided performance dashboard allows visualization of host metrics, CPU flame graphs, and other performance-related data. Follow these steps to deploy locally and see instructions on using the dasboard:
+
+    ```bash
+    make -f builder/Makefile.performance dashboard-start
+    ```
+
+    ```bash
+    make -f builder/Makefile.performance dashboard-stop
+    ```
+
+3. **Benchmarking:** Before submitting significant code changes, consider running benchmarks to assess their impact on Tracee's performance. (Details on specific benchmarking tools or scripts used within the Tracee project should be added here. If there are existing benchmarks, provide instructions on how to run them. If not, suggest a methodology).
```

Comment:
Is this a note for you @ShohamBit ?
`(Details on specific benchmarking tools or scripts used within the Tracee project should be added here. If there are existing benchmarks, provide instructions on how to run them. If not, suggest a methodology).`